### PR TITLE
Add docs for policygen.

### LINKED ISSUE #9: Add usage guide for policygen
Similar to https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/tree/master/cmd/tfengine#usage

### PR DESCRIPTION
**Pull request by xingao267**
This PR also moves template dir info to input config and format other
README files.

Fixes #9 
#10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,17 @@
 This repository contains a suite of tools that can be used to manage key areas
 of your GCP organization.
 
-- Deploy
-- Monitor
-- Audit
+-   Deploy
```
Comment (umairidris) [inline README.md:6]: I mentioned this in Martin's PR too but this is actually in violation of markdown rule https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md030. Please use https://github.com/DavidAnson/vscode-markdownlint or https://github.com/igorshubovych/markdownlint-cli

File: cmd/policygen/README.md [hunk 2]
```diff
@@ -5,12 +5,66 @@ Status: Early Access Program
...
+To deploy GCP Organization Policy Constraints, execute the following commands:
+
+```shell
+cd $OUTPUT_DIR/gcp_organization_policies
+terraform init
+terraform plan
+terraform apply
+```
+
+#### Policy library Constraints
```
Comment (MartinPetkov) [inline cmd/policygen/README.md:64]: Are all the files generated by policygen compatible with both Forseti and Terraform Validator?

File: examples/policygen/config.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+template_dir: ../../templates/policygen
```
Comment (MartinPetkov) [inline examples/policygen/config.yaml:15]: This and org_id are required, could you leave a comment marking them as such?

Comment (xingao267) [reply examples/policygen/config.yaml:16]: Done

File: README.md (hunk 1 above)
Comment (xingao267) [reply README.md]: Done.

**Pull request merged**